letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -210,16 +210,28 @@ def deploy_cert(self, domain, cert_path, key_path,
                 "Unable to find cert and/or key directives")
 
         logger.info("Deploying Certificate to VirtualHost %s", vhost.filep)
-
-        # Assign the final directives; order is maintained in find_dir
-        self.aug.set(path["cert_path"][-1], cert_path)
-        self.aug.set(path["cert_key"][-1], key_path)
-        if chain_path is not None:
-            if not path["chain_path"]:
-                self.parser.add_dir(
-                    vhost.path, "SSLCertificateChainFile", chain_path)
-            else:
-                self.aug.set(path["chain_path"][-1], chain_path)
+        logger.debug("Apache version is %s",
+                     ".".join(str(i) for i in self.version))
+
+        if self.version < (2, 4, 8):
```

Comment:
I guess one could argue that we should nudge people towards doing things the "right" way for Apache 2.4.  But conversely we could argue that if someone sets up automated scripts calling `letsencrypt install` on an Apache 2.2 system, they shouldn't break just because Apache was upgraded to 2.4.  So I think add `or (cert_path and not fullchain_path)`